#53 (frontend): fix new position drop down button

Yifang-Zhang opened the pull request.

dingtaoliu committed:
TAPP-57: fixed deletePosition function in positions/action.js to make a delete request to the api

Commit:
TAPP-53 fix new position drop down button

chrislessard reviewed (changes requested):
The code looks good to me. The commit message should change, however, please follow the format specified here: https://github.com/uoft-tapp/tapp/issues/39

As a suggestion, you could write: 

fix(frontend): Clear the new position drop down button

This commit fixes a bug with the "new postion" drop down button, in which the dropdown persists after redirection.

Commit:
Merge pull request #60 from uoft-tapp/TAPP-57

TAPP-57: fixing delete button functionality on edit position popup

Commit:
fix(frontend): Clear the new position drop down button

This commit fixes a bug with the "new postion" drop down button, in which the dropdown persists after redirection.

Yifang-Zhang commented:
fix(frontend): Clear the new position drop down button

This commit fixes a bug with the "new postion" drop down button, in which the dropdown persists after redirection.
